assessmentResults/src/commonMain/kotlin/org/sagebionetworks/assessmentmodel/navigation/Direction.kt

```diff
@@ -0,0 +1,20 @@
+package org.sagebionetworks.assessmentmodel.navigation
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+enum class Direction {
+    /**
+     * Move forward through the assessment.
+     */
+    Forward,
+    /**
+     * Move backward through the assessment.
+     */
+    Backward,
+    /**
+     * Exit the assessment early. If this direction indicator is set, then the entire assessment run should end.
+     */
+    Exit,
+    ;
```

Comment:
Note: Not sure how much it matters but swift defaults for the serialized name is lowercase so these won't match.